docs/sphinx/quickstart.rst

```diff
@@ -2,7 +2,7 @@
     Copyright (C) 2018 Mikael Simberg
 
     SPDX-License-Identifier: BSL-1.0
-    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    Distributed under the |boost| Software License, Version 1.0. (See accompanying
```

Comment:
This one should stay as `Boost` as it's not actually rendered in the documentation.
```suggestion
    Distributed under the Boost Software License, Version 1.0. (See accompanying
```